shaded/client/pom.xml

```diff
@@ -123,9 +125,9 @@
                 </excludes>
               </artifactSet>
               <filters>
-                <!-- Make sure no source code included -->
+                <!-- Exclude source code included in non-source jar -->
                 <filter>
-                  <artifact>*:*</artifact>
```

Comment:
The filter applies to both normal and source jars, unfortunately. Thus, the original filter causes the shaded source jar to be empty, since all of the input sources are filtered out.

I left the `zookeeper-jute` exclusion since they incorrectly include sources in the normal jar. So we get the positive of not having them in the shaded jar, with the negative that shaded source jar won't have them.

Reply:
Thanks for the explanation. I'm good with not having `zookeeper-jute` in shaded source jar.